config/settings.yml

```diff
@@ -95,13 +95,18 @@ scheme_filters_enabled?: <%= ENV['ENV'] == 'demo' %>
 # Feature flag to enable ability to set user email notification user preferences.  If true, external
 # users will be able to set preference to say whether they want email notifications of outstanding messages
 # on their claim
-email_notification_enabled?: true
+email_notification_enabled?: false
```

Comment:
is this needed? remove if not needed or leave as is, in a separate commit. we need a "Flag cleanup" ticket generally. I started looking at some myself

Reply:
I have created https://dsdmoj.atlassian.net/browse/CBO-1444 for tidying feature flags generally, and will have a look at whether we can remove the email_nofication_enabled? flag in a separate pr.

Reply:
We are still using email_notification_enabled here https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/9fa5b40b78fcc434b1a9ddaf99fe7a4070871ddd/app/views/external_users/admin/external_users/index.html.haml and here https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/dd477c91c67785010c31e266bdd66160b72d2fdb/app/views/external_users/admin/external_users/_form.html.haml